packages/web-components/fast-foundation/src/tree-view/tree-view.ts

```diff
@@ -147,16 +147,32 @@ export class TreeView extends FoundationElement {
             case keyArrowLeft:
                 if (e.target && this.isFocusableElement(e.target as HTMLElement)) {
                     const item = e.target as HTMLElement;
-                    if (item instanceof TreeItem && item.childItemLength() > 0) {
+
+                    if (
+                        item instanceof TreeItem &&
+                        item.childItemLength() > 0 &&
+                        item.expanded
+                    ) {
                         item.expanded = false;
+                    } else if (
+                        item instanceof TreeItem &&
+                        item.parentElement instanceof TreeItem
+                    ) {
+                        TreeItem.focusItem(item.parentElement);
                     }
```

Comment:
Left arrow on either a child node or a closed node moves focus to its parent (if one exists).